VimeoNetworking/Sources/Request+Cache.swift

```diff
@@ -32,12 +32,10 @@ public extension Request
     /// Generates a unique cache key for a request, taking into account endpoint and parameters
     var cacheKey: String
     {
-        var cacheKey = "cached" + self.path
+        let url = NSURL(string: self.path)
+        let urlPath = url?.path ?? ""
```

Comment:
Aren't `self.path` and `urlPath` the same thing? Do we even need to construct an NSURL?